From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Paolo Sabatino <paolo.sabatino@gmail.com>
Date: Tue, 6 Jan 2026 18:13:52 +0100
Subject: [ARCHEOLOGY] rockchip: other fixes and cleanups for current/edge
 kernels

> X-Git-Archeology: > recovered message: > * DFI driver fix to let DDR3 rk322x handle DMC driver
> X-Git-Archeology: > recovered message: > * edge kernel: removal for Gigabit Ethernet DMA
> X-Git-Archeology: > recovered message: > workaround for rk3288
> X-Git-Archeology: > recovered message: > * reinstate CONFIG_DEBUG_FS=y with both current/edge kernels
> X-Git-Archeology: - Revision 911a160980b50d1ba0ec6367c123ea6319c6455d: https://github.com/armbian/build/commit/911a160980b50d1ba0ec6367c123ea6319c6455d
> X-Git-Archeology:   Date: Tue, 06 Jan 2026 18:13:52 +0100
> X-Git-Archeology:   From: Paolo Sabatino <paolo.sabatino@gmail.com>
> X-Git-Archeology:   Subject: rockchip: other fixes and cleanups for current/edge kernels
> X-Git-Archeology:
---
 drivers/devfreq/event/rockchip-dfi.c | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/drivers/devfreq/event/rockchip-dfi.c b/drivers/devfreq/event/rockchip-dfi.c
index 111111111111..222222222222 100644
--- a/drivers/devfreq/event/rockchip-dfi.c
+++ b/drivers/devfreq/event/rockchip-dfi.c
@@ -161,9 +161,8 @@ static int rockchip_dfi_ddrtype_to_ctrl(struct rockchip_dfi *dfi, u32 *ctrl)
 			ddrmon_ver);
 		return -EOPNOTSUPP;
 	default:
-		dev_err(&dfi->edev->dev, "unsupported memory type 0x%X\n",
-			dfi->ddr_type);
-		return -EOPNOTSUPP;
+		*ctrl = 0;
+		break;
 	}
 
 	return 0;
-- 
Armbian

